src/Core/Module/ModuleManagerInterface.php

```diff
@@ -50,6 +50,8 @@ public function disableMobile(string $name): bool;
 
     public function reset(string $name, bool $keepData = false): bool;
 
+    public function delete(string $name): bool;
```

Comment:
BC Break? 🤯 Nitpick but...

Reply:
Whaaaaat? It's a new function :D

Reply:
Of course it's a bc break, but you can get rid of it 
see `\PrestaShop\PrestaShop\Core\Util\String\StringModifierInterface`